lib/random.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Picks a random number between min and max (inclusive).
+ *
+ * @param {number} min
+ * @param {number} max
+ * @returns {number}
+ */
+export const pickRandomNumber = (min, max) => {
+  min = Math.ceil(min)
+  max = Math.floor(max)
```

Comment:
```suggestion
```

We can trust the caller that min < max

Reply:
I've added asserts to make sure correct inputs are passed down https://github.com/CheckerNetwork/walrus-checker/pull/15/commits/88be5847bf3bd177e780f1a6ce8bc83b4e588e51

Reply:
```suggestion
```

We still don't need these two lines though